dotnet/Gherkin/CucumberMessages/Types/Step.cs

```diff
@@ -5,23 +5,23 @@ namespace Gherkin.CucumberMessages.Types;
 public class Step
 {

```

Comment:
According to the schema, the only required elements of a Step are: Location, Keyword, Text, and Id.

Reply:
what about KeywordType? should it be optional? as I see - this is only 1 difference

Reply:
According to the schema, KeywordType, Docstring, and DataTable are optional.

Reply:
fixed